iac/main.tf

```diff
@@ -0,0 +1,57 @@
+# Copyright 2024 The Sigstore Authors
+# SPDX-License-Identifier: Apache-2.0
+
+provider "google" { project = var.project_id }
+provider "google-beta" { project = var.project_id }
+provider "ko" { docker_repo = "gcr.io/${var.project_id}" }
+
+// Create a network with several regional subnets
+module "networking" {
+  source  = "chainguard-dev/common/infra//modules/networking"
```

Comment:
same here - can we use OSS for this?

Reply:
it is oss :) https://github.com/chainguard-dev/terraform-infra-common

Reply:
@bobcallaway if you don't like i can redo manually

Reply:
ok... must have been my jetlag that I didn't link it to being a terraform provider ... nm then :)

Reply:
https://registry.terraform.io/modules/chainguard-dev/common/infra/latest